library/src/swarm.ts

```diff
@@ -0,0 +1,44 @@
+import { SwarmMessages } from "./messages/swarm-messages";
+import { createSwarmMessages } from "./messages/swarm-messages.factory";
+import { SessionId } from './model/general.types';
+import { LocalStorage } from './services/local-storage';
+import { BzzLink } from './services/bzz-link'
+import { PostageBatch } from "./services/postage-batch";
+import { Web2HelperContent } from "./services/web2-helper.content";
+
+/**
+ * Interface of the Swarm browser extension
+ * This class can be used inside of a web page or service worker script
+ */
+export class Swarm {
+  private messages: SwarmMessages
+
+  public bzzLink: BzzLink
+
+  public localStorage: LocalStorage
+
+  public postageBatch: PostageBatch
+
+  public web2Helper: Web2HelperContent
+
+
+  /**
+   *
+   * @param extensionId The Swarm extension ID
+   */
+  constructor(extensionId: string = 'afpgelfcknfbbfnipnomfdbbnbbemnia') {
```

Comment:
`Type string trivially inferred from a string literal, remove type annotation.eslint[@typescript-eslint/no-inferrable-types](https://github.com/typescript-eslint/typescript-eslint/blob/v4.33.0/packages/eslint-plugin/docs/rules/no-inferrable-types.md)`
```suggestion
  constructor(extensionId = 'afpgelfcknfbbfnipnomfdbbnbbemnia') {
```